example/src/screens/calendarsList.js

```diff
@@ -1,10 +1,10 @@
 import React, {Component} from 'react';
 import {CalendarList} from 'react-native-calendars';
 
-export default class CalendarList extends Component {
+export default class HorizontalCalendarList extends Component {
```

Comment:
This is not the horizontal CalendarList example, why did you change the name?

Reply:
my linter is complaining about the duplicate class names, but I'll disable it